src/System.Memory/ref/System.Memory.cs

```diff
@@ -72,6 +79,11 @@ namespace System
         public Span<T> Slice(int start, int length) { throw null; }
         public T[] ToArray() { throw null; }
         public bool TryCopyTo(Span<T> destination) { throw null; }
+        public ref struct Enumerator
+        {
+            public bool MoveNext() { throw null; }
+            public ref T Current { get { throw null; } }
```

Comment:
What about adding Reset?
http://referencesource.microsoft.com/#mscorlib/system/collections/ienumerator.cs

Reply:
Reset is legacy and isn't needed by foreach.